acvm/src/pwg/brillig.rs

```diff
@@ -13,6 +13,8 @@ use crate::{
 
 use super::{directives::insert_witness, get_value};
 
+const MIN_BRILLIG_REGISTERS: usize = 16;
```

Comment:
Should this say "MAX_BRILLIG_REGISTERS`instead of `MIN` then?

We would not want any constants in here that are explicitly linked to the public VM, if we need this functionality in the future, we can add a way to set this when we instantiate Brillig. We may be able to just say that the thing that is generating the Brillig code should just do register allocation to ensure that the max number of registers is X.